components/Collections/src/MDCCollectionViewController.m

```diff
@@ -399,16 +399,12 @@ - (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
 
 #pragma mark - <UICollectionViewDelegate>
 
-- (BOOL)collectionView:(UICollectionView *)collectionView
-    shouldHighlightItemAtIndexPath:(NSIndexPath *)indexPath {
-  if ([_styler.delegate respondsToSelector:@selector(collectionView:hidesInkViewAtIndexPath:)]) {
-    return ![_styler.delegate collectionView:collectionView hidesInkViewAtIndexPath:indexPath];
-  }
```

Comment:
Are you suggesting keeping the shouldHighlight... and returning YES each time, but then in didHighlight keeping the proposed changes?